dtbase/core/sensors.py

```diff
@@ -241,14 +241,14 @@ def get_datatype_by_measure_name(measure_name, session=None):
     )
     result = session.execute(query).fetchall()
     if len(result) == 0:
-        raise ValueError("No sensor measure called '{measure_name}'")
+        raise ValueError("No sensor measure named '{measure_name}'")
```

Comment:
I think this needs an "f" before the string, i.e.  (f"No sensor measure named '{measure_name}'")

Reply:
Done!